components/inputs/README.md

```diff
@@ -109,6 +110,7 @@ The `<d2l-input-text>` element is a simple wrapper around the native `<input typ
 
 **Properties:**
 
+- `aria-haspopup` (String):  indicates that the input has a popup menu
```

Comment:
This was added because `dropdown` is adding it to `d2l-input-text` but it isn't being brought through to the `input`.

`title` was added in order to add more context/direction for screenreader users (perhaps there was a better way to do that? I saw it used in an example)